packages/core/src/config/load.ts

```diff
@@ -58,3 +55,15 @@ function findConfig() {
   }
   return undefined;
 }
+
+export async function getRawConfigContent(configPath = findConfig()) {
```

Comment:
add unit test

Reply:
this function was removed, now use the previous version of loadConfig